src/Backend/Request.php

```diff
@@ -25,21 +24,6 @@ public function __construct(ClientInterface $client)
     }
 
     /**
-     * @param string $url
-     * @param string $method
-     * @param array  $options
-     *
-     * @return RequestInterface
-     */
-    protected function createRequest($url, $method = 'GET', $options = [])
```

Comment:
The method is protected and could be used by someone who extends this class => BC break 

If this was private, everything would be okay

Also it hasn't been marked as internal @liayn